Makefile

```diff
@@ -138,7 +138,7 @@ ginkgo:
 
 localintegration: varlink_generate test-binaries
 	ginkgo -v -cover -flakeAttempts 3 -progress -trace -noColor test/e2e/.
-	sh test/varlink/run_varlink_tests.sh
+	#bash test/varlink/run_varlink_tests.sh
```

Comment:
should this part be dropped?

Reply:
Yes this looks like it removes the test?

Reply:
For now, yes

Reply:
I also didn't manage to get it working using the go:1.8 image, it seems Python varlink is broken somehow as it doesn't work with Python2 or Python3.

Could we add a comment though saying these tests are currently disabled?

Reply:
done